Backpack-SwiftUI/Tests/BottomSheet/BPKBottomSheetTests.swift

```diff
@@ -0,0 +1,57 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import XCTest
+import SwiftUI
+@testable import Backpack_SwiftUI
+
+class BPKBottomSheetTests: XCTestCase {
+    func test_fullSize_bottom_sheet() {
+        assertSnapshot(
+            BPKBottomSheetContent(
+                isPresented: .constant(true),
+                isClosable: false,
+                contentMode: .fullSize,
+                content: {
+                    Spacer()
+                    BPKText("This this bottom sheet content")
+                    Spacer()
+                })
+            .frame(width: 300, height: 300)
+        )
+    }
+    
+    func test_regular_bottom_sheet() {
+        assertSnapshot(
+            BPKBottomSheetContent(
```

Comment:
I think instead of testing the Content, we should test the bottom sheet itself. 
Can you change these tests to do something like `BPKButton().closableBottomSheet(...)` or `BPKButton().bottomSheet(...)`

From this tests, is not clear that the drag indicator is there or how's the alignment on the sheet

Reply:
I am working on this one right now.